docs/mobile/events.md

```diff
@@ -45,12 +45,12 @@ The event name MUST be `device.app.lifecycle`.
 
 ### Android
 
-<!-- semconv android.lifecycle.events -->
+<!-- semconv android.lifecycle.events(full) -->
```

Comment:
have you been beaten up by #305 here?

Reply:
Hum not sure what you mean? Is that the correct linked issue? I needed to add `full` because now things are in the registry and it doesn't render them :(